bounties_api/std_bounties/client_helpers.py

```diff
@@ -168,6 +169,11 @@ def notify_slack(sc, channel, event, msg):
 
     return True
 
+def bounty_url_for(bounty_id):
+    #url = reverse('std_bounties:bounty-detail', args=[bounty_id])
+    url = 'https://beta.bounties.network/bounty/v1/{}'.format(bounty_id)
```

Comment:
@zoek1 can you have it pull an env var from settings.py?  I'll add it - just call it deploy_url

Reply:
done! :+1: